test/linter/test-obsolete.js

```diff
@@ -0,0 +1,103 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import chalk from 'chalk-template';
+import { Logger } from '../utils.js';
+import bcd from '../../index.js';
+const { browsers } = bcd;
+
+const exceptions = [
+  'api/VRDisplay.json',
+  'api/VREyeParameters.json',
+  'api/VRFieldOfView.json',
+  'api/VRPose.json',
+  'api/Window.json',
+  'http/headers/cache-control.json',
+  'http/headers/feature-policy.json',
+  'svg/attributes/presentation.json',
+  'svg/elements/view.json',
+];
+
+function neverImplemented(support) {
+  for (const s in support) {
+    let data = support[s];
+    if (!Array.isArray(data)) data = [data];
+    for (const d of data) if (d.version_added) return false;
+  }
+  return true;
+}
+
+function implementedAndRemoved(browsers, support) {
+  for (const browser in support) {
+    let data = support[browser];
+    if (!Array.isArray(data)) data = [data];
+    for (const d of data) {
+      if (!d.version_removed) return false;
+      if (
+        d.version_removed &&
+        new Date(
+          browsers[browser].releases[d.version_removed].release_date,
+        ).getTime() >
+          new Date().getTime() - 2 * 365 * 24 * 60 * 60 * 1000
```

Comment:
Could we actually hard-code a date?  Performing math to calculate two years difference is more than alright, but I wouldn't want to have the lint suddenly fail on `main` and cause test failures in others' PRs!